src/universal/containers/Logout/LogoutContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes, Component} from 'react';
+import {connect} from 'react-redux';
+import {push} from 'react-router-redux';
+import {show} from 'universal/modules/notifications/ducks/notifications';
+import {removeAuthToken} from 'universal/redux/authDuck';
+
+const logoutSuccess = {
+  title: 'Tootles!',
+  message: 'You\'ve been logged out successfully.',
+  level: 'success'
+};
+
+@connect()
+export default class LogoutContainer extends Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired
+  };
+
+  componentWillMount() {
+    const {dispatch} = this.props;
+    dispatch((thunkDispatch) => Promise.all([
```

Comment:
that's interesting... and we're sure it's due to the persisting?
Regardless, I've known that eventually I need to come up with a whitelist of actions that trigger a persist because this persist-after-every-action is just hogwash. I'll PR `redux-storage` and get this crackin.